spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -116,6 +116,27 @@
       end
     end
 
+    context 'locked out session' do
+      let(:locked_at) { Time.zone.now }
+      let(:user) { create(:user, :fully_registered) }
+
+      before do
+        allow(subject).to receive(:session_bad_password_count_max_exceeded?).
+          and_return(true)
+        allow(subject).to receive(:clear_session_bad_password_count_if_window_expired).
+          and_return(nil)
+        allow(IdentityConfig.store).to receive(:max_bad_passwords_window_in_seconds).and_return(600)
+      end
+
+      it 'renders an error letting user know they are locked out for a period of time' do
```

Comment:
Can we add a test to show that the "less than 1 minute" versus "X minutes" works as intended?